vpcmanager/src/main/java/com/futurewei/vpcmanager/comm/exception/CacheException.java

```diff
@@ -0,0 +1,20 @@
+package com.futurewei.vpcmanager.comm.exception;
```

Comment:
Cache and logging are common across microservices. Let us move out of the vpcmanager directory.